src/main/java/io/jenkins/plugins/audit/listeners/UserLogListener.java

```diff
@@ -93,19 +96,20 @@ protected void loggedIn(@Nonnull String username) {
      *
      * @param username name or ID of the user who logged out.
      */
-     @Override
-     protected void loggedOut(@Nonnull String username) {
-         String currentTime = Instant.now().toString();
-         Logout logout = LogEventFactory.getEvent(Logout.class);
+    @Override
+    protected void loggedOut(@Nonnull String username) {
+        String currentTime = Instant.now().toString();
+        Logout logout = LogEventFactory.getEvent(Logout.class);
 
-         RequestContext.setNodeName("master");
-         RequestContext.getRequestId();
-         RequestContext.setIpAddress(Stapler.getCurrentRequest().getRemoteAddr());
-         logout.setUserId(username);
-         logout.setTimestamp(currentTime);
-         logout.logEvent();
-         RequestContext.clear();
-     }
+        RequestContext.setNodeName("master");
+        RequestContext.getRequestId();
+        RequestContext.setIpAddress(Stapler.getCurrentRequest().getRemoteAddr());
+        RequestContext.setUserId(username);
+        logout.setUserId(username);
+        logout.setTimestamp(currentTime);
+        logout.logEvent();
+        RequestContext.clear();
```

Comment:
I mean, this is why you had to reset the data in the first place. The request context shouldn't be cleared until the end of the request, and that would likely be part of the servlet filter task.

Reply:
hmh... so maybe i'll just move all the `RequestContext` method call codes out ..

Reply:
Yeah, that sounds like a good idea to me. :)